test/styles/RRR.R

```diff
@@ -3,19 +3,23 @@
 ### Function declarations
 
 ## 1
-fun <- function(argument1,
-                argument2) {
-    body
+{
+    fun <- function(argument1,
+                    argument2) {
+        body
+    }
 }
 
 ## 2
-function(
-    argument1,
-    argument2
-    )
+{
+    function(
+             argument1,
```

Comment:
Granted, this override makes more sense for `ess-offset-arguments` than for `ess-offset-arguments-newline`. In the latter situation, it's clear that the author wants to indent its arguments backward.

I could have `ess-indent-function-declaration` only override `ess-offset-arguments` and not the `newline` variant. But we could also consider that the purpose of this setting in this case is to create a style incentive to always indent function arguments at opening parenthesis, so it's probably fine this way.